railties/test/application/test_test.rb

```diff
@@ -345,6 +346,44 @@ def self.load_schema!
       assert_unsuccessful_run "models/user_test.rb", "SCHEMA LOADED!"
     end
 
+    def test_actionable_command_line_error
+      rails "generate", "scaffold", "user", "name:string"
+      app_file "config/environments/test.rb", <<-RUBY
+        Rails.application.configure do
+          config.actionable_command_line_errors = true
+        end
+      RUBY
+      app_file "config/initializers/thor_yes.rb", <<-RUBY
+        Rails::Command::Base.class_eval <<-MORERUBY
+          def yes?(statement, color = nil)
+            raise ArgumentError unless statement == "Run pending migrations? [Yn]"
+            true
+          end
```

Comment:
This solution isn't ideal, but this was the best we could come up with. Mocking `Thor::LineEditor#readline` / `$stdin` within the generated test app doesn't seem to work. Does anyone have a better idea how we could test this?